docs/source/hardware_and_software_configuration/configuring/configuring_dual_hubs/configuring-dual-hubs.rst

```diff
@@ -1,25 +1,19 @@
-Using Two Expansion Hubs
-========================
+Adding an Expansion Hub
+=======================
 
 Introduction
 ~~~~~~~~~~~~
 
 A single REV Robotics Control or Expansion Hub has a limited amount of input/output
 (I/O) ports available. In some instances, you might want to use more
 devices than there are ports available. For these instances you might
-need to connect a second Expansion Hub to your first Hub to add more I/O
+need to connect an Expansion Hub to your first Hub to add more I/O
 ports.
 
-This document describes how to connect and configure two Expansion Hubs
-for use in the FIRST Tech Challenge. Note that the FIRST Tech Challenge
-Competition Manual limits the maximum number of
-Expansion Hubs on a single robot to two.
-
-**Important Note:** This document describes the process for setting up a
-smartphone Robot Controller with two Expansion Hubs. Control Hubs have a
-reserved address, so you do not need to worry about an Expansion Hub's
-address when it is the only Expansion Hub connected to a Control Hub.
-However, the process for physically connecting and configuring them is the same.
+This document describes how to connect and configure an additional Expansion
+Hub for use in the FIRST Tech Challenge. Note that the FIRST Tech Challenge
+Competition Manual limits the maximum number of Control or Expansion Hubs on
+a single robot to two. See rule R701 for the full details.
```

Comment:
Rephrase to not point to s specific rule number. Rule numbers may shift year to year.  

Suggest something like:

> Check the Robot Construction rules in the [Competition Manual](https://ftc-resources.firstinspires.org/file/ftc/game/manual) for details.